app/models/lookup_key.rb

```diff
@@ -125,6 +126,12 @@ def path=(v)
     write_attribute(:path, using_default ? nil : v)
   end
 
+  def reject_invalid_lookup_values(attributes)
+    attributes[:match].empty? || attributes[:match].match(VALUE_REGEX).blank? ||
```

Comment:
Is this covered by a functional or unit test anywhere so we know that ordinary nested attributes will get accepted?  e.g. if the regexp changed to match nothing at all, will we know?

A test for the case you're fixing (where the matcher attribute's nil) might be useful.